app/views/batches/_form.html.erb

```diff
@@ -1,4 +1,53 @@
 <%= form_with(model: batch, local: true, id: dom_id(batch), class: 'mt-3', data: { 'reflex-root': '#mappers' }) do |form| %>
+<% if flash[:csv_lint] %>
+  <article class="flash message is-danger is-small mt-3">
+    <div class="message-header">
+      <p>Uploaded CSV is invalid</p>
+    </div>
+    <div class="message-body">
+      <p class="content">
+	Consult <a href="https://github.com/Data-Liberation-Front/csvlint.rb/#errors">csvlint's errors documentation</a> for more info on what the different errors mean.</p>
+	<ul>
+	<% flash[:csv_lint].split('|||').each do |err| %>
+	  <li><%= err %></li>
+	<% end %>
+	</ul>
+      <p>&nbsp;</p>
+      <p>If you are getting "unknown_error", some things to check for might include:</p>
+      <ul>
+	<li>Line breaks within a CSV cell may be breaking the CSV structure</li>
+      </ul>
+    </div>
+  </article>
+<% end %>
+
+<% if flash[:csv_too_long] %>
+  <article class="flash message is-danger is-small mt-3">
+    <div class="message-header">
+      <p>Uploaded CSV has more than <%= csv_row_limit %> rows</p>
+    </div>
+    <div class="message-body">
+      <p class="content">The maximum number of rows this tool can import from a single CSV is <%= csv_row_limit %>. Please break your CSV file into multiple smaller files and create a separate batch for each.</p>
+    </div>
+  </article>
+<% end %>
+
+<% if flash[:batch_config] %>
```

Comment:
@mark-cooper - I can't find where this gets set anywhere. It looks like batch config errors get added to batch.errors with value "is invalid JSON" and displayed here instead of via that flash message: 

![image](https://user-images.githubusercontent.com/450570/203160632-27236487-d3fe-4e22-8fb6-813d876da709.png)

Reply:
Removed this in [subsequent commit](https://github.com/collectionspace/collectionspace-csv-importer/pull/191/commits/652192e047d34325c48cba2ce8b2fe1588ea0032)